src/components/TaskList/TaskList.js

```diff
@@ -45,6 +68,23 @@ const TaskList = (props) => {
                     })
                 }
             </div>
+
+        </div>
+    );
+};
+
+const MenuOptions = ({id}) => {
+    const DeleteTask = async (id) => {
```

Comment:
Please change method name "DeleteTask" to "DeleteTaskList"
Also delete all tasks inside that tasklist
PS: I'd forgot to mention that